src/material/schematics/ng-add/schema.json

```diff
@@ -34,10 +34,18 @@
       "x-prompt": "Set up global Angular Material typography styles?"
     },
     "animations": {
-      "type": "boolean",
-      "default": true,
-      "description": "Whether Angular browser animations should be set up.",
-      "x-prompt": "Set up browser animations for Angular Material?"
+      "type": "string",
+      "default": "enabled",
+      "description": "Whether Angular browser animations should be included.",
```

Comment:
Could we elaborate here on which components strictly require an animations module to function?

Reply:
We could, but then the option will be somewhat long and the list is bound to go out of date.

Reply:
I don't think the option is really useful without communicating that somehow, though. I wouldn't expect someone to go looking through the source to figure out where it's necessary, and letting someone omit animations just to then get an error when they try to use a component isn't a great experience.

Reply:
I feel like that would be too much text to fit in a multi-select list in the command line. We _could_ generate a list of components using animations from the source code and publish it as a URL on material.angular.io that we then link to from the schematic. It might take a while to do it though.

Reply:
Something like that would probably be useful. I've been meaning to write an "About" page for material.angular.io for a while and a good section there would be "Animations supports"